lib/rbac/filterer.rb

```diff
@@ -445,7 +445,12 @@ def scope_targets(klass, scope, rbac_filters, user, miq_group)
         # if subclasses of MetricRollup or Metric, use the associated
         # model to derive permissions from
         associated_class = rbac_class(scope)
+        if associated_class.respond_to?(:scope_by_tenant?) && associated_class.scope_by_tenant?
+          scope_associated_ids = scope_to_tenant(associated_class, user, miq_group).ids
+        end
+
         filtered_ids = calc_filtered_ids(associated_class, rbac_filters, user, miq_group)
+        filtered_ids = [scope_associated_ids, filtered_ids].compact.flatten
```

Comment:
I cannot pass it to the first parameter because scope tenant condition in `scope_tenant_filter` would be lost, so I added it as new parameter.
Is this ok for now ?

For now we need to make 1 query to determine `scope_tenant_filter.ids`, in follow up PR I would like to look if it is possible to merge it to main `scope`  and skip this query. 

thanks for review @kbrock